internal/controllers/user_devices_controller.go

```diff
@@ -1580,14 +1569,10 @@ func (udc *UserDevicesController) PostMintDevice(c *fiber.Ctx) error {
 				return err
 			}
 
-			return client.MintVehicleAndSDign(requestID, contracts.MintVehicleAndSdInput{
-				ManufacturerNode: makeTokenID,
-				Owner:            realAddr,
-				AttrInfoPairsVehicle: []contracts.AttributeInfoPair{
-					{Attribute: "Make", Info: deviceMake},
-					{Attribute: "Model", Info: deviceModel},
-					{Attribute: "Year", Info: deviceYear},
-				},
+			return client.MintVehicleAndSdWithDeviceDefinitionSign(requestID, contracts.MintVehicleAndSdWithDdInput{
+				ManufacturerNode:    makeTokenID,
+				Owner:               realAddr,
+				DeviceDefinitionId:  userDevice.DeviceDefinitionID,
```

Comment:
I think Lorran is going to merge in a version of this with attributes soon.

Reply:
Can we flag this by environment, actually? It would be nice to still be able to push this to main.